tests/test_models.py

```diff
@@ -164,19 +164,13 @@ def test_update_a_record(self):
         logging.debug(record)
         record.create()
         self.assertIsNotNone(record.product_id)
-        # Change it an save it
-        record.reorder_quantity = 15
-        record.quantity = 10
-        record.restock_level = 2
-        original_id = record.product_id
-        record.update()
-        self.assertEqual(record.product_id, original_id)
-        self.assertEqual(record.reorder_quantity, 15)
-        self.assertEqual(record.quantity, 10)
-        self.assertEqual(record.restock_level, 2)
-        records = Inventory.all()
-        self.assertEqual(len(records), 1)
-        self.assertEqual(records[0].product_id, original_id)
-        self.assertEqual(records[0].reorder_quantity, 15)
-        self.assertEqual(records[0].quantity, 10)
-        self.assertEqual(records[0].restock_level, 2)
+
+        request_body = record.serialize()
+        request_body["quantity"] = 10
+        request_body["restock_level"] = 2
+        request_body["reorder_quantity"] = 15
+        new_data = Inventory()
+        new_data.deserialize(request_body)
+        record.update(new_data)
```

Comment:
shouldn't we call the URL with a PUT method instead of calling update in models?

Reply:
Yes, we are calling the put method from a test in `test_routes`. Since this was a `test_models` do you think we can call the `update` method directly?

Reply:
oh sorry I thought this function was written in test_routes. my bad